lnbits/core/templates/core/account.html

```diff
@@ -139,7 +155,7 @@ <h4 class="q-my-none">Account Settings</h4>
                   :src="'{{ static_url_for('static', 'images/google-logo.png') }}'"
                 ></q-img>
               </q-avatar>
-              <div>Google</div>
+              <div><span v-text="$t('google')"></span></div>
```

Comment:
Do you expect that "Google" is called something else in some languages?

If not, I think we should not translate this string. Same for "GitHub".

Reply:
Maybe in Pirate :)
Reverted in this commit https://github.com/lnbits/lnbits/pull/2171/commits/6785846d1cfbe720287cd53fb11348d61251aafc